test/__mocks__/ts-dpop.ts

```diff
@@ -1 +1,4 @@
-export const verify = jest.fn((): string => 'http://alice.example/card#me');
+import type { VerifySolidIdentityFunction } from 'ts-dpop';
+
+export const verifySolidIdentityFunction = jest.fn().mockResolvedValue({ aud: 'solid', exp: 1234, iat: 1234, iss: 'example.com/idp', webid: 'http://alice.example/card#me' });
```

Comment:
And also I would
```suggestion
const verifySolidIdentityFunction = jest.fn().mockResolvedValue({ aud: 'solid', exp: 1234, iat: 1234, iss: 'example.com/idp', webid: 'http://alice.example/card#me' });
```

such that it has the same interface as the original.